android/service/src/main/kotlin/net/mullvad/mullvadvpn/service/ForegroundNotificationManager.kt

```diff
@@ -90,10 +92,18 @@ class ForegroundNotificationManager(
         }
 
     fun showOnForeground() {
-        service.startForeground(
-            TunnelStateNotification.NOTIFICATION_ID,
-            tunnelStateNotification.build()
-        )
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
```

Comment:
This should probably be its own PR

Reply:
Agree: https://github.com/mullvad/mullvadvpn-app/pull/5473

Reply:
Retracted